<launch>

<node pkg="image_work" name="hw7node" type="hw7node.py"/>
<node pkg="image_processing_hw" name="image_pub_all" type="image_pub_all.py"/>
<node pkg="rqt_image_view" name="image_view_cropped" type="rqt_image_view" args="/image_cropped" />
<node pkg="rqt_image_view" name="image_view_white" type="rqt_image_view" args="/image_white" />
<node pkg="rqt_image_view" name="image_view_yellow" type="rqt_image_view" args="/image_yellow" />

</launch>
